src/common/cmdparse.cc

```diff
@@ -679,4 +679,23 @@ bool cmd_getval(const cmdmap_t& cmdmap,
   return false;
 }
 
+bool cmd_getval_compat_cephbool(
+  const cmdmap_t& cmdmap,
+  const std::string& k, bool& val)
```

Comment:
nit, might want to return an `optional<bool>`. and pass `k` using `std::string_view`.

Reply:
the signature looks like this to match the other ``cmd_getval`` methods... i agree optional would be nicer if designing from scratch.

Reply:
okay. i will create a follow-up PR to fix them all.